executor/common.h

```diff
@@ -527,7 +527,6 @@ static void loop(void)
 				break;
 #endif
 			event_timedwait(&th->done, /*{{{CALL_TIMEOUT}}}*/);
-			break;
```

Comment:
The inner loop over threads finds the first available thread to schedule the call. Once we schedule the call, we break out of the loop because we don't want to schedule the same call on another thread.
The current code looks correct to me.
If you still this this break is wrong, try to come up with a runtest that demonstrates the bug:
https://github.com/google/syzkaller/tree/master/sys/test/test

Reply:
Thanks for explanation, that does make sense.